.github/actions/create-kubewarden-cluster/action.yaml

```diff
@@ -0,0 +1,64 @@
+name: "Setup Kubewarden cluster"
+description: "Create a K3d cluster with Kubewarden installed with Telemetry infrastructure installed"
+author: "Kubewarden"
+inputs:
+  controller-image-repository:
+    description: "Define the controller container image repository"
+    type: string
+    required: false
+    default: ""
+  controller-image-tag:
+    description: "Define the controller container image tag"
+    required: false
+    default: ""
+  load-local-image:
+    description: "Load the image used in the deployment from local artifact"
+    required: false
+    default: false
+runs:
+  using: "composite"
+  steps:
+      - uses: AbsaOSS/k3d-action@v1.5.0
+        name: "Create Kubernetes cluster"
+        with:
+          cluster-name: "e2e-test-cluster"
+      - name: "Download local image"
+        if: ${{ inputs.load-local-image }}
+        uses: actions/download-artifact@v2
+        with:
+          name: kubewarden-controller-image-${{inputs.controller-image-tag}}
+          path: /tmp
```

Comment:
From the assets. I did that to avoid publishing a container image in the registry which is not ready for use. 

[See an example](https://github.com/jvanz/kubewarden-controller/actions/runs/1500495322)